server/deploy_test.go

```diff
@@ -204,7 +204,7 @@ func TestGetDatabaseMatrixFromDeploymentSchedule(t *testing.T) {
 				},
 			},
 			"world",
-			"",
+			"{{DB_NAME}}",
```

Comment:
Still add a case for empty db name template?

Reply:
Yeah, I will handle the empty string case later with more testing.